concepts/05 Widgets/Scheduler/030 Appointments/020 Add Appointments/020 API.md

```diff
@@ -76,4 +76,90 @@ To add an appointment to the [data source](/api-reference/10%20UI%20Widgets/dxSc
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="dataSource"
+            :current-date="currentDate"
+        />
+        <DxButton
+            text="Add"
+            :on-click="onClick"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxScheduler from 'devextreme-vue/scheduler';
+
+    export default {
+        components: {
+            DxScheduler
+        },
+        data() {
+            return {
+                dataSource: [{
+                    text: "Planning",
+                    startDate: new Date(2016, 4, 25, 9, 00),
+                    endDate: new Date(2016, 4, 25, 9, 30)
+                }, 
+                // ...
+                ],
+                currentDate: new Date(2016, 4, 25),
+                onClick: (e) => {
```

Comment:
In Vue, functions should be declared in [the `methods` object](https://v1.vuejs.org/guide/events.html).